public/index.html

```diff
@@ -4,6 +4,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://www.lactame.com/lib/ml/4.0.0/ml.min.js"></script>
```

Comment:
We should probably snapshot this and other libraries at some point so that we can serve them off our own domain, avoiding potential firewall issues..

Reply:
added to the to do list